src/components/Header/index.tsx

```diff
@@ -98,6 +102,7 @@ export default () => {
         )}
       </HeaderPanel>
       <MaintainAlert />
+      {activate && <NodeAlert />}
```

Comment:
I would strongly recommend using `ternary expression` because I've seen `0` rendered several times when the type is not property guarded.